### ISSUE TITLE
cice.setup broken for cases and tests since PR #395

### ISSUE THREAD

**Issue by phil-blain** [CICE-Consortium/CICE#405]
Since PR #395 was merged it appears that `cice.setup` can't create cases or individual tests, in both cases I get a syntax error in the `cice.setup` script:

~~~shelll
$ ./cice.setup --test smoke --mach cesium  -s diag1,debug --testid myid
 
./cice.setup:
else: endif not found.
~~~

I did not have time to find where is the error yet, just letting you know @apcraig.

We should add a call to `cice.setup --case` and `cice.setup --test` in the `.travis.yml` such that breakages like this do not go unnoticed in the future.

**Comment by phil-blain** [comment]
I guess this would be high or extreme priority...

**Comment by apcraig** [comment]
Thanks @phil-blain, I'll have a look and get a fix in this evening.

**Comment by apcraig** [comment]
I just committed the fix #406 and added some tests to travis that would trap this in the future.  Sorry about the error, I guess this'll happen occasionally.  Glad we caught it fairly quickly.  I will close this.
